### ISSUE TITLE
Create exec_dir if it does not exist

### ISSUE THREAD

**Issue by jgaida** [oar-team/cigri#24]
I got the following error trying to create the directory using the JDL prologue:
  The job exited because of a working directory error. NFS problem on the cluster?;

**Comment by bzizou** [comment]
Yes, it is a good idea to create the exec_dit if it does not exist.
Two solutions:
- Using the OAR api by posting an empty file into that directory (e.g. POST /media/exec_dir/dummy_file), but which module can do that?
- Adding an inconditionnal "mkdir -p" at the top of the prologue.
The second solution is very simple but requires that there's a prologue. For the moment, the prologue is optional. It could be mandatory...

**Comment by jgaida** [comment]
We can make it optional by adding a "create_dir" option in the JDL
